src/theme/styles/colours.ts

```diff
@@ -104,6 +108,8 @@ export const DARK_COLOURS = {
   textActive1: '',
   textDisabled: '#31323E',
 
+  icon: '#8D8DA980',
```

Comment:
Why not use rgba() ?

Reply:
Because all other colors are defined as `hex`. Wanted to keep the same pattern.
I don't mind changing, though. Do you prefer that?

Reply:
haha the perfect "designer vs developer" situation - hex is much easier for us design-lacking devs to copy and paste, but i totally agree rgba is better for manipulating colours, and adds a nice opacity alpha. In this case it's def us keeping a standard format. 

We may want to consider switching entirely to rgba @alfetopito @anxolin

Reply:
I think here @biocom should be the last authority. Whatever he feels more comfortable with. 
I personally don't modify ever a color by hand, I use a color picker. So hex is fine for me.

Reply:
> haha the perfect "designer vs developer" situation - hex is much easier for us design-lacking devs to copy and paste, 

Besides copy and paste, does it provide you any context on opacity? Looking at:
`rgb(141 141 169 / 20%)`
`rgb(141 141 169 / 50%)`

I know I'm dealing with a single color, but with a different alpha (opacity) setting. Looking at a matching HEX example:
`#8d8da933`
`#8d8da980`

Can you honestly tell what the exact opacity is?

Reply:
I personally cannot, my hex math is not the best =P

I would then be in favor of making all colors RGB[A], for no other reason than I can search for a color and know whether that's defined already somewhere or not. 
For example when I need a color and I don't know whether we have the same exact one, I do a text search for the hex value.

If that's alright, I can make a PR transforming the colors from hex to rgb[a]

Reply:
My take would be:

1. Where ALPHA colors are not needed, just use a 6 digit HEX.
2. Where ALPHA is used, use the alpha version.

Ideally these colors are only defined in the `src/theme/styles/colours.ts` file, so searching for a color most of times would be by variable name?

That being said, while working on a front-end ticket myself, I was thinking to try to exclusively only use 6 digit HEXes and instead try to use the `opacity` CSS property where possible. 

As you know in some cases that won't work (e.g. needing transparency only on a `border` property without effecting the whole element with `opacity`).